src/deploy-tes-on-azure/scripts/helm/templates/tes-deployment.yaml

```diff
@@ -66,6 +66,8 @@ spec:
               value: {{ .Values.identity.resourceId }}
             - name: BatchScheduling__UsePreemptibleVmsOnly
               value: {{ .Values.config.batchScheduling.usePreemptibleVmsOnly | quote}}
+            - name: BatchScheduling__TaskMaxWallClockTime
```

Comment:
`TaskMaxWallClockTimeDays` would be more clear it seems?

Reply:
Yes